assets/queries/terraform/azure/mariadb_server_georedundant_backup_disabled/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	mdb := input.document[i].resource.azurerm_mariadb_server[name]
+
+	not common_lib.valid_key(mdb, "geo_redundant_backup_enabled")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_mariadb_server[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("'azurerm_mariadb_server[%s].geo_redundant_backup_enabled' is defined and not null", [name]),
+		"keyActualValue": sprintf("'azurerm_mariadb_server[%s].geo_redundant_backup_enabled' is undefined or null", [name]),
+		"searchLine": common_lib.build_search_line(["resource", "azurerm_mariadb_server", name], []),
+	}
+}
+
+CxPolicy[result] {
+	mdb := input.document[i].resource.azurerm_mariadb_server[name]
+
+	mdb.geo_redundant_backup_enabled == false
```

Comment:
I think we could add `common_lib.valid_key(mdb, "geo_redundant_backup_enabled")` check, before checking if `geo_redundant_backup_enabled` is false...
```suggestion
  common_lib.valid_key(mdb, "geo_redundant_backup_enabled")
	mdb.geo_redundant_backup_enabled == false
```

Reply:
Once we want that "geo_redundant_backup_enabled" be false, supposedly the attribute is already defined because to be false the attribute must exist, so we don't need it.